website/docs/r/media_services.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_media_services"
+sidebar_current: "docs-azurerm-resource-media-media-services"
+description: |-
+  Manages a Media Services account.
+---
+
+# azurerm_media_services
+
+Manages a Media Services account.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "testrg" {
+  name     = "amstestrg"
+  location = "westus"
+}
+
+resource "azurerm_storage_account" "testsa" {
+  name                     = "amstestsa"
+  resource_group_name      = "${azurerm_resource_group.testrg.name}"
+  location                 = "${azurerm_resource_group.testrg.location}"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_media_services" "ams" {
+
+        name = "amstest"
+        location = "${azurerm_resource_group.testrg.location}"
+        resource_group_name = "${azurerm_resource_group.testrg.name}"
+		
+        storage_account {
+				id = "${azurerm_storage_account.testsa.id}"
+				is_primary = true
+		}
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the MariaDB Server. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the MariaDB Server. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `storage_account` - (Required) A `storage_account` block as defined below. Multiple storage_account arguments can be provided, but only one can be marked as `is_primary`=true.
```

Comment:
Could we `quote` true, this also might be better moved into a note block. WDYT?

Reply:
I `quoted` "true" and moved this into `## Note`